Gemfile

```diff
@@ -49,6 +49,7 @@ group :development, :test do
   gem 'better_errors',              '~> 2.1', '>= 2.1.1'
   gem 'annotate',                   '~> 2.6', '>= 2.6.10'
   gem 'pry-rails',                 '0.3.4'
+  gem 'pry-nav'
```

Comment:
Sorry about that!  I should have looked around the Gemfile first...
It seems `pry-nav` is not actively maintained, so this wouldn't have been an optimal addition.  I'll take it out of this PR!